tests/e2e/lib/page-helper.js

```diff
@@ -5,102 +5,24 @@ import config from 'config';
 /**
  * WordPress dependencies
  */
-import { pressKeyWithModifier } from '@wordpress/e2e-test-utils';
 /**
  * Internal dependencies
  */
 import logger from './logger';
 import { execSyncShellCommand } from './utils-helper';
 
-/**
```

Comment:
One of the reasons for these custom methods was to log every test action, to see what's going on. You mentioned that there is `DEBUG=pw:api` env variable that we can use to achieve similar behavior. I'd like to hear your take on this, and maybe how would you approach it.

Reply:
Indeed those methods were logging actions, if/where they were used. Playwright's auto-wait introduces the risk of not using those methods in a consistent way.
I also believe the logged information was only useful in case you're debugging, otherwise it's polluting the log. In case of failure, the failing action is logged anyway, with element locator if relevant. With Playwright, if you want to debug, running the tests with `DEBUG=pw:api` will give the same result, but with more information and in a consistent way.

I've added the option under the E2E_DEBUG, so there's a single debug toggle. So now, `E2E_DEBUG=true` will also trigger `pw:api` logging. 

Also, I'm not very familiar with Winston, but I'm assuming there as way to filter pw:api log entries. Then have a different debug file with only those entries. Or have a file with everything and a clean one without those entries. There are more ideas.